bin/fin

```diff
@@ -3900,11 +3900,6 @@ run_cli ()
 {
 	local env_str=""
 
-	local HOST_UID=$(id -u)
-	local HOST_GID=$(id -g)
-
-	[[ -f "$CONFIG_ENV" ]] && set -a; source "$CONFIG_ENV";
```

Comment:
@lmakarov I'm curious why we don't want to have this in here? This would allow for variables from the home `docksal.env` file. Such as storing a Terminus Token or Acquia Credentials.

Reply:
By eliminating this line then this doesn't test doesn't work proplery

```
 	echo "TEST=1234" >> $HOME/.docksal/docksal.env
 	run fin rc -T -e TEST 'echo \$TEST'
 	[[ "$output" == "1234" ]]
 	unset output
```

Reply:
I apologize I misread the chain of events and found in the `RUNTIME STARTS HERE` is where the file is sourced.

Reply:
The global `docksal.env` is already sourced at the very beginning of runtime in `fin` (https://github.com/docksal/docksal/blob/develop/bin/fin#L5000), so `SECRET_` variables will be passed without issues.

`HOST_UID` and `HOST_UID` used to be in `load_configuration`. I moved them to the runtime section as well.

Tests look good.